### PR TITLE
Discussion polish

### PR DESCRIPTION
**Pull request by agapow**
Some light polishing of first part of discussion section: rewording for flow, to avoid repetition of words, etc.

### CODE REVIEW

**Comment by agitter** [comment]
@agapow The old commits were wrapped up in this pull request again.  Did that still happen after cloning a fresh copy of the repository?

**Comment by agapow** [comment]
No, still using the old one. I'll clone a new one now.

File: sections/06_discussion.md [hunk 1]
```diff
@@ -212,7 +221,7 @@ to recognize those who promote an ecosystem of rigorous sharing and analysis
 The sharing of high-quality, labeled datasets will be especially valuable.  In
 addition, researchers who invest time to preprocess datasets to be suitable for
 deep learning can make the preprocessing code (e.g. Basset
-[@tag:Kelley2016_basset] and variationanalysis [@tag:Torracinta2016_deep_snp])
```
Comment (agitter) [inline sections/06_discussion.md]: Revert this please.  `variationanalysis` in lower case is the method name.

File: sections/06_discussion.md [hunk 2]
```diff
@@ -281,10 +292,10 @@ machine learning algorithms. When trained on such datasets, deep learning
 models are particularly prone to overfitting due to their high representational
 power. However, transfer learning techniques also known as domain adaptation
 enable transfer of extracted patterns between different datasets and even
-domains. This approach can be described in a 2-step process:
-(1) training a model for the base task, and (2) reusing the trained model
+domains. This approach consists of training a model for the base task, and
+subsequently reusing the trained model
 for the target problem in hand. The first step allows a model to take advantage
-of larger amount of data, labels, or both, to extract better feature
+of larger amount of data and/or labels to extract better feature
```
Comment (agitter) [inline sections/06_discussion.md]: Not your edit, but should this read "advantage of a larger"?

File: sections/06_discussion.md [hunk 3]
```diff
@@ -307,10 +318,10 @@ distant domains such as natural images and various modalities of biological
 imaging. Because learnt patterns are represented in deep neural networks in a
 layer-wise hierarchical fashion, this issue is usually addressed by fixing an
 empirically chosen number of layers that preserve generic characteristics
-of both training and target datasets. Then, the model is fine-tuned by
+of both training and target datasets. The model is then fine-tuned by
 re-training multiple networks' top layers on the specific dataset in order to
-re-learn domain-specific high level concepts. For example, see fine-tuning for
-radiology image classification [@tag:Rajkomar2017_radiographs].
+re-learn domain-specific high level concepts (e.g.fine-tuning for
```
Comment (agitter) [inline sections/06_discussion.md]: I believe you've been adding commas after e.g. elsewhere

**Comment by agitter** [comment]
@agapow we can merge this before you clone if you'd like so you don't lose these changes.

File: sections/06_discussion.md [hunk 4]
```diff
@@ -54,46 +54,52 @@ For domain-specific models, we previously described approaches for the
...
+training datasets by splitting a single labeled example into multiple examples.
+For example, an
 image can easily be rotated, flipped, or translated and retain its label
 [@doi:10.1101/095794].  3D MRI and 4D fMRI (with time as a dimension) data can
-be decomposed into sets of 2D images [@doi:10.1101/070441]. This greatly expands
-the number of training examples in domains where only hundreds of subjects are
-available but artificially treats images from the same volume as independent
+be decomposed into sets of 2D images [@doi:10.1101/070441]. This can greatly
+expand
+the number of training examples
```
Comment (agapow) [inline sections/06_discussion.md]: I thought it was implicit: the idea is to compensate for a small training set. So, reiterating that the set was small just lengthens the text and makes it more labourious to read.

File: sections/06_discussion.md (hunk 1 above)
Comment (agapow) [inline sections/06_discussion.md]: Ha! This caught me out.

File: sections/06_discussion.md (hunk 2 above)
Comment (agapow) [inline sections/06_discussion.md]: It should. Will fix.

File: sections/06_discussion.md (hunk 3 above)
Comment (agapow) [inline sections/06_discussion.md]: Not me. Cleaned some of those up.

**Comment by agapow** [comment]
Will attend to these changes and then leave it to you.

File: sections/06_discussion.md (hunk 1 above)
Comment (agitter) [inline sections/06_discussion.md]: Note to self when naming methods: don't use all lower case compound words

File: sections/06_discussion.md (hunk 4 above)
Comment (agitter) [inline sections/06_discussion.md]: Okay, works for me

File: sections/06_discussion.md (hunk 3 above)
Comment (agitter) [inline sections/06_discussion.md]: Oh, now I see.  I think this is missing a space though.

**Comment by agitter** [comment]
I'm working on resolving merge conflicts in #368.  The interpretation section edits here will be lost because the text has completely changed.

**Pull request merged**